index.js

```diff
@@ -727,7 +788,13 @@ function addManualTaskCheckUi() {
             <div id="objective-task-manual-check" class="extensionsMenuExtensionButton fa-regular fa-square-check"/></div>
             Manual Task Check
         </div>`);
+    $('#extensionsMenu').prepend(`
```

Comment:
Please use `getWandContainer()`. This is a fallback mechanism for those who haven't updated to 1.12.2 yet

Reply:
Nvm, fixed it on my own. Everything else looks good.